src/main/java/com/exasol/adapter/dialects/rewriting/ImportIntoTemporaryTableQueryRewriter.java

```diff
@@ -44,20 +46,30 @@ public ImportIntoTemporaryTableQueryRewriter(final SqlDialect dialect,
         this.connectionFactory = connectionFactory;
     }
 
+    /*
+     * If selectListDataTypes is provided then use columns description from there otherwise infer columns description
+     * from query.
+     */
     @Override
-    protected String generateImportStatement(final String connectionDefinition, final String pushdownQuery)
-            throws SQLException {
-        final String columnDescription = this.createImportColumnsDescription(pushdownQuery);
-        return "IMPORT INTO (" + columnDescription + ") FROM JDBC " + connectionDefinition + " STATEMENT '"
+    protected String generateImportStatement(final String connectionDefinition,
```

Comment:
I would prefer if you used method overloading instead of an `if`. This way you can keep the interface backward compatible while being also able to deprecate the old interface. I know that this affects a number of methods in the chain, but I think it's worth the effort.